### ISSUE TITLE
React-Native-Navigation gist out dated.

### ISSUE THREAD

**Issue by 3210jr** [wix/react-native-notifications#198]
Hey,
We are unable to use any other notification library because of the choice to use react-native-navigation. By following the hard to follow gist link on the homepage, the app doesn't even start any more. Crashes on launch.

Is there a way we can get an official demo version of how these two should work?

Notifications are an essential part to any application and the two packages are really amazing on their own, its just using them together that is hard.

Any help appreciated!

React Native: 0.53
react-native-navigation: 1.1.409
react-native-notifications: 1.1.19

**Comment by gjvpaet** [comment]
Any update on this? My app also crashes on launch.

**Comment by EliSadaka** [comment]
I followed the installation instructions in the wiki and [here](https://github.com/wix/react-native-notifications/issues/37) but my app is also now crashing on launch. Is this package not compatible with newer versions of React Native or is something else perhaps causing this issue?

**Comment by pribeh** [comment]
Has anyone tried the most recent version 1 of react native navigation and gotten it to work with react native notifications?

**Comment by hugows** [comment]
@pribeh @EliSadaka @gjvpaet @3210jr Any progress on this?

**Comment by EliSadaka** [comment]
@hugows We moved over to using the overlay functionality in React Native Navigation v2 and react-native-firebase (since our notifications are handled through Firebase) to implement our notifications instead.

**Comment by sorin-f-wiredelta** [comment]
Is it even worth trying to get this to work with RNNv2 ?
